pinot-common/src/test/resources/sql_queries.list

```diff
@@ -817,3 +817,19 @@ select COUNT(*) from baseballStats where DIV(numberOfGames,10) = 100
 select mapKeys(mapField) from baseballStats where DIV(numberOfGames,10) = 100
 select mapKey(mapField,k1) from baseballStats where DIV(numberOfGames,10) = 100
 select mapKey(mapField,k1) from baseballStats where mapKey(mapField,k1) = 'v1'
+SELECT count(*) FROM Foo GROUP BY VALUEIN(mv_col, 10790, 16344) LIMIT 10
```

Comment:
The equivalent SQL (preserving old behavior of the pql query) will be 
```
SELECT VALUEIN(mv_col, 10790, 16344), count(*) FROM Foo GROUP BY VALUEIN(mv_col, 10790, 16344) ORDER BY count(*) DESC LIMIT 10
```
Am I right?

Reply:
Correct. I missed adding this here for parser test. But had added these earlier in query execution tests.

Reply:
done